src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml

```diff
@@ -867,9 +868,13 @@
     </Style>
 
     <!--  Zoom fade text  -->
-    <Style x:Key="SZoomFadeText" TargetType="{x:Type TextBlock}">
+    <Style
+        x:Key="SZoomFadeBase_Animation"
+        TargetType="{x:Type FrameworkElement}">
```

Comment:
TargetType was changed here to support more element types ? not just Textblock ?

Reply:
yeah can be reused.